inc/templates.php

```diff
@@ -466,10 +466,14 @@ function parse_config_from_registry( array $component ) {
 		'text'   => 'is_string',
 	];
 
-	foreach ( $registered['config'] ?? [] as $key => $atts ) {
+	// Loop through registered config.
+	foreach ( ( $registered['config'] ?? [] ) as $key => $atts ) {
+
+		// This value has been set and a sanitize callback exists.
 		if (
-			isset( $component['config'][ $key ] ) &&
-			( call_user_func( $type_callbacks[ $atts['type'] ], $component['config'][ $key ] ) )
+			isset( $component['config'][ $key ] )
+			&& isset( $type_callbacks[ $atts['type'] ] )
```

Comment:
Not sure what you are trying to solve for here. Are you trying to make `type` an optional value when registering config values, or did you run across an unsupported type that broke here? If the latter, I'd prefer that we add additional supported types/callbacks above rather than skipping any that we can't validate.

Reply:
I had a typo of `arrays` instead of `array` and it was throwing an error. This ensures that the callback exists before we try to use it.

Reply:
In that case, seems like we need better error handling instead of working around errors. Otherwise, this seems to have worked as it should have, no?

Reply:
The only issue here is that it was attempting to call a `$type_callbacks[]` key that didn't exist. This adds a check to ensure it exists.

The actual calling of the method works fine.

Reply:
Or are you suggesting something like a wp_die printing out an error message suggesting the developer check their "type" key? As I continue to think about it, that makes more sense to me.

Reply:
Updated in https://github.com/alleyinteractive/wp-irving/pull/163/commits/08b040a7e730a4f74a8237ccb631527cab0f50fb